examples/05-Q3D/Q2D_Armoured_Cable.py

```diff
@@ -0,0 +1,359 @@
+"""
+Q2D: Cable parameter identification
+---------------------------------------------------
+This example shows how you can use PyAEDT to:
+ - create a Q2D design using the Modeler primitives and importing part of the geometry
+ - set up the whole simulation
+ - link the solution to a Simplorer design
+ - reference: https://www.luxingcable.com/low-voltage-cables/4-core-armoured-power-cable.html
```

Comment:
```suggestion
This example shows how you can use PyAEDT to perform these tasks:

 - Create a Q2D design using the Modeler primitives and importing part of the geometry.
 - Set up the entire simulation.
 - Link the solution to a Simplorer design.

 For cable information, see `4 Core Armoured Power Cable <https://www.luxingcable.com/low-voltage-cables/4-core-armoured-power-cable.html>`_
```
Google dev style guide says to introduce a list with a complete sentence. You need a blank line before the first list item and after the last one. Links should have display text that matches where you are taken.